src/Corrade/Utility/Debug.cpp

```diff
@@ -59,13 +108,16 @@ template<Debug::Color c> Debug::Modifier Debug::colorInternal() {
         if(!noSpaceBefore) debug << Debug::nospace;
         debug << code;
         if(noSpaceBefore) debug << Debug::nospace;
```

Comment:
You are just backing up restoring the state of  `InternalFlag::NoSpaceBeforeNextValue` which is getting reset in `debug << code`.
In the case of `CORRADE_TARGET_WINDOWS` this flag does not get touched and therefor does not need this backup/restore code.